services/QuillLMS/app/models/question.rb

```diff
@@ -149,4 +154,37 @@ def rematch_type
   private def stored_as_array(key)
     data[key].class == Array
   end
+
+  private def validate_sequences
+    return if !data.present? || !data.is_a?(Hash)
+    parse_and_validate(data['incorrectSequences'])
+    parse_and_validate(data['focusPoints'])
+  end
+
+  private def parse_and_validate(sequences)
+    return if !sequences.present?
```

Comment:
`sequences.blank?`.  Also, it's good to add a line after a line [break](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/EmptyLineAfterGuardClause)) after guard clauses to signal control flow